apps/livreBatch/livre_batch.py

```diff
@@ -204,7 +204,7 @@ def _calc_ranges(i):
         # Submit job(s) for range(s)
         idx = 1
         for sub_range in ranges:
-            idx += self._submit_jobs_for_range(idx, sub_range[0], sub_range[1])
+            idx += self._submit_jobs_for_range(idx, sub_range[0], sub_range[1] + 1)
```

Comment:
Now I get it, it wasn't obvious:

    ideal_range = set(range(start_frame, end_frame))

returns a closed range set, so you have to add back the last one.